### PR TITLE
feat: add dls info to divisions

### PR DESCRIPTION
**Pull request by svetavedmed**
edenlabllc/ehealth.api#4553

### CODE REVIEW

File: __schema__/divisions.graphql [hunk 1]
```diff
@@ -76,6 +78,10 @@ type Division implements Node {
   legalEntity: LegalEntity!
   "Division working hours."
   workingHours: DivisionWorkingHours
+  "verification acording to State Service of Ukraine for drug control"
```
Comment (progsmile) [inline __schema__/divisions.graphql]:
```suggestion
  "Verification according to State Service of Ukraine for drug control."
```

File: __schema__/divisions.graphql [hunk 2]
```diff
@@ -10,6 +10,8 @@ input DivisionFilter {
   databaseId: UUID
   "Division name."
   name: String
+  "verification acording to State Service of Ukraine for drug control"
```
Comment (progsmile) [inline __schema__/divisions.graphql]:
```suggestion
  "Verification according to State Service of Ukraine for drug control."
```

File: __schema__/divisions.graphql [hunk 3]
```diff
@@ -76,6 +78,10 @@ type Division implements Node {
   legalEntity: LegalEntity!
   "Division working hours."
   workingHours: DivisionWorkingHours
+  "verification acording to State Service of Ukraine for drug control"
+  dlsVerified: Boolean
+  "Id in the registery of State Service of Ukraine for drug control"
+  dlsID: UUID!
```
Comment (progsmile) [inline __schema__/divisions.graphql]:
```suggestion
  dlsId: UUID!
```

Comment (progsmile) [inline __schema__/divisions.graphql]:
```suggestion
  dlsId: UUID!
```

Comment (svetavedmed) [reply __schema__/divisions.graphql]: no, it could be null or UUID. Not all divisions could be verified according to this registry

**Pull request merged**